app.js

```diff
@@ -1,6 +1,48 @@
+const { request, response } = require("express");
 const express = require("express");
 const app = express();
+const Ticket = require("./ticketModel");
 
 app.use(express.static("client/build"));
 
+app.get("/api/tickets", (request, response) => {
+  const searchText = request.query.searchText;
+  Ticket.find({})
+    .then((allTickets) => {
+      if (searchText) {
+        const filteredArr = allTickets.filter((ticket) =>
+          ticket.title.toLowerCase().includes(searchText.toLowerCase())
+        );
+        response.status(200).json(filteredArr);
+      } else {
+        response.status(200).json(allTickets);
+      }
+    })
+    .catch((err) => response.status(404).send(err, "Not Found"));
```

Comment:
There should be a `500` error here, not `404`.
`404` message is unnecessary here.